graphqlmetrics/CHANGELOG.md

```
@@ -4,6 +4,12 @@ Binaries are attached to the github release otherwise all images can be found [h
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [0.37.0](https://github.com/wundergraph/cosmo/compare/graphqlmetrics@0.36.2...graphqlmetrics@0.37.0) (2025-12-15)
+
+### Features
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)